container.mk

```diff
@@ -0,0 +1,46 @@
+
+all: clean-tmp test ## Cleans environment, builds docker image and runs tests
+
+boot_database:
+	until bin/rake boot:database TEST_ENV_NUMBER=8 ; do \
+		sleep 1 ; \
+		echo -n "." ; \
+	done
+	if [ "x$$DB" = "xoracle" ]; then \
+		echo "Waiting for 60 seconds for the DB to be ready" ; \
+		sleep 60 ; \
+	fi
+
+clean-tmp: ## Removes temporary files
+	-@ $(foreach dir,$(TMP),rm -rf $(dir);)
+
+dnsmasq_set:
+	echo "nameserver $$DNSMASQ_PORT_53_TCP_ADDR" > resolv.conf.dnsmasq && sudo cp /etc/resolv.conf /etc/resolv.conf.dist && sudo cp resolv.conf.dnsmasq /etc/resolv.conf
```

Comment:
Make should not depend on sudo. Like `make install` you do `sudo make install`, not let `make install` run sudo.

Reply:
ah, yes, good point, thanks - I'll make the necessary changes